myriad-executor/build.gradle

```diff
@@ -1,3 +1,11 @@
+apply plugin: 'application'
+
+mainClassName = "com.ebay.myriad.Main"
+
+run {
+    systemProperty "myriad.config", "build/resources/main/myriad-config-default.yml"
```

Comment:
Also, myriad-config-default.yml is only used by the scheduler and not the executor. Not sure if this is needed in the executor gradle.